### ISSUE TITLE
Counters dont update after changing in settings

### ISSUE THREAD

**Issue by TGCFabian** [Caeden117/CountersPlus#32]
**Describe the Bug**
Counters+ does work unless you start tweaking, while moving stuff around, it wont change on screen, restarting the game, turning the mod on or off, making a new config, nothing works

**Attach or Link Files**
To help me solve this issue, I might need access to more then what you say or provide.

Attached the config file
[CountersPlus.zip](https://github.com/Caeden117/CountersPlus/files/3829125/CountersPlus.zip)

The error for me happened while on stream, thus not being able to give a --verbose output, but i will attach logs
The date was the date of posting, same with time

Will include a extra one before stream

[2019.10.21.21.47.36.log.gz](https://github.com/Caeden117/CountersPlus/files/3829129/2019.10.21.21.47.36.log.gz)
[_latest.log](https://github.com/Caeden117/CountersPlus/files/3829130/_latest.log)

**Probable Culprit**
Updated all mods to latest, counters+ is on beta version 1.7.0-beta1

**Screenshots or Video**
https://www.twitch.tv/videos/506661989
Basically the beginning of the stream is me trying to figure out the issue, and after a while i made this report

**Additional context**

Nop

**Comment by Caeden117** [comment]
[It's been fixed. I am currently doing last minute preparations then Counters+ 1.7.0 should be making its way onto Mod Assistant. Try this for the time being](https://github.com/Caeden117/CountersPlus/files/3829137/Counters%2B.zip)
